examples/jetpack_compose/app/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="cm.ben.android.bazel.compose.example">
```

Comment:
Shall we change the package name to something closer to the Kotlin rules' package?

Reply:
I'm ok with it.